docs/onboarding/onboarding-feature.md

```diff
@@ -0,0 +1,173 @@
+| ordering               | 20                                      |
+| primary-button-label   | juno_onboarding_sign_in_positive_button |
+| secondary-button-label | juno_onboarding_sign_in_negative_button |
+
+## Notification card
+| Attribute              | Value                                                   |
+|------------------------|---------------------------------------------------------|
+| card-type              | notification-permission                                 |
+| title                  | juno_onboarding_enable_notifications_title_nimbus       |
+| body                   | juno_onboarding_enable_notifications_description_nimbus |
+| link-text (optional)   | juno_onboarding_default_browser_description_link_text   |
+| image-res              | ic_notification_permission                              |
+| image-is-illustration  | true                                                    |
+| ordering               | 30                                                      |
+| primary-button-label   | juno_onboarding_enable_notifications_positive_button    |
+| secondary-button-label | juno_onboarding_enable_notifications_negative_button    |
+| secondary-button-label | juno_onboarding_sign_in_negative_button                 |
+
+# Available resources
+
+## String resources
+All existing app strings are available to Experimenter. [See the full list here](https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/app/src/main/res/values/strings.xml).
+Free text may also be used instead of an image resource.
+
+## Image resources
+
+### Campaign - Default (Illustrations)
+| Card type               | Resource ID                |
+|-------------------------|----------------------------|
+| default-browser         | ic_onboarding_welcome      |
+| sync-sign-in            | ic_onboarding_sync         |
+| notification-permission | ic_notification_permission |
+
+### Campaign - Lifestyle (Photographic)
+
+| Card type               | Resource ID                                                                                                                                                               |
+|-------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| default-browser         | [onboarding_default_browser](https://raw.githubusercontent.com/mozilla-mobile/firefox-android/main/fenix/app/src/main/res/drawable-xxhdpi/onboarding_default_browser.webp)| 
+| sync-sign-in            | [onboarding_sync](https://raw.githubusercontent.com/mozilla-mobile/firefox-android/main/fenix/app/src/main/res/drawable-xxhdpi/onboarding_sync.webp)                      |
+| notification-permission | [onboarding_notification](https://raw.githubusercontent.com/mozilla-mobile/firefox-android/main/fenix/app/src/main/res/drawable-xxhdpi/onboarding_notification.webp)      |
+
+# Creating an experiment
```

Comment:
I'd put this section up higher: the readers of this doc are interested in creating an experiment first and foremost. Once they know how to do that, then add the nuance and the nitty details.